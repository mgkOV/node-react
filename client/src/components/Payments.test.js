import React from 'react';
import { shallow } from 'enzyme';
import Payments from './Payments';

test('Payments renders correctly', () => {
  const component = shallow(<Payments />);
  expect(component).toMatchSnapshot();
});
