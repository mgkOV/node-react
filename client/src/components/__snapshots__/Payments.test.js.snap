// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payments renders correctly 1`] = `
<ReactStripeCheckout
  ComponentClass="span"
  amount={500}
  className="StripeCheckout"
  description="$5 for 5 email credits"
  label="Pay With Card"
  locale="auto"
  name="Emailiy"
  reconfigureOnUpdate={false}
  stripeKey="pk_test_SRZSjVPE8XfncILGQAhDTmnV"
  token={[Function]}
  triggerEvent="onClick"
>
  <button
    className="btn"
  >
    Add credits
  </button>
</ReactStripeCheckout>
`;
